src/Components/SmartComponents/Reports/Common/tablePage.js

```diff
@@ -114,10 +114,29 @@ const tablePage = ({ data, page, intl, header, type, isReportDynamic = false })
                 </Text>
             );
 
+            const affectedCell = (
+                <View style={[styles.bodyCell, styles.cveCell]}>
+                    <View>
```

Comment:
Why is the `<View>` nested? :thinking:

Reply:
Nesting Views are fine. REF: https://react-pdf.org/components#view

Reply:
I have revisited the PR and looks like you were right. There is no need to have another nested View component. I thought it was needed for proper alignment. Have a look if this is correct indeed. Thanks